Integrations/integration-Symantec_Messaging_Gateway.yml

```diff
@@ -95,7 +95,15 @@ script:
         }
         login_url = 'login.do'
         login_response = http_request('post', login_url, cookies=cookies, data=data)
-        jsession = login_response.cookies.get_dict()['JSESSIONID']
+
+        # check if status is valid
+        login_response.raise_for_status()
+
+        #if JSESSIONID doesn't exist - creds may be invalid
+        try:
+            jsession = login_response.cookies.get_dict()['JSESSIONID']
```

Comment:
This could fail for other reasons, please check if the cookies contain `JSESSIONID`